### PR TITLE
Docs/update links

### PR DESCRIPTION
**Pull request by CalMacCQ**
Updating the examples link to go to the examples README instead of just the folder.

### CODE REVIEW

File: pytket/docs/index.rst [hunk 1]
```diff
@@ -16,8 +16,6 @@ Windows. To install, run
 
 If you have issues installing ``pytket`` please visit the `installation troubleshooting <https://cqcl.github.io/tket/pytket/api/install.html>`_ page.
 
-To use ``pytket``, you can simply import the appropriate modules into your python code or in an interactive Python notebook. We can build circuits directly using the ``pytket`` interface by creating a blank circuit and adding gates in the order we want to apply them.
```
Comment (cqc-alec) [inline pytket/docs/index.rst:19]: Why delete this?

File: pytket/docs/index.rst [hunk 2]
```diff
@@ -90,7 +88,7 @@ Licensed under the `Apache 2 License <http://www.apache.org/licenses/LICENSE-2.0
     
     Manual <https://cqcl.github.io/pytket/manual/index.html>
     Extensions <https://cqcl.github.io/pytket-extensions/api/index.html>
-    Example notebooks <https://github.com/CQCL/pytket/tree/main/examples>
+    Example notebooks <https://github.com/CQCL/pytket/tree/main/examples#pytket-examples>
```
Comment (cqc-alec) [inline pytket/docs/index.rst:91]: This link is still not working.

**Review by cqc-alec — CHANGES_REQUESTED** [review]

File: pytket/docs/index.rst (hunk 2 above)
Comment (CalMacCQ) [reply pytket/docs/index.rst:91]: I think its just the #pytket examples heading that changed

File: pytket/docs/index.rst (hunk 1 above)
Comment (CalMacCQ) [reply pytket/docs/index.rst:19]: The text seemed a bit redundant to me. If preferred I can leave it for now and update this is a fresh PR.

File: pytket/docs/index.rst (hunk 2 above)
Comment (CalMacCQ) [reply pytket/docs/index.rst:91]: works now at least

**Review by cqc-alec — APPROVED** [review]

**Comment by CalMacCQ** [comment]
I don't think theres much point in merging this and updating the link until the issues with deployment on the pytket repo are resolved.

The link that is in the API docs currently will work as before and link to the github repo.

**Pull request closed without merging**